sql/plan.go

```diff
@@ -136,6 +136,9 @@ type planNode interface {
 	Next() bool
 	// Err returns the error, if any, encountered during iteration.
 	Err() error
+	// ExplainPlan fills in name and description for the node and returns a list
+	// of child nodes.
+	ExplainPlan(name, description *string) []planNode
```

Comment:
I'm not sure. I actually started off returning strings and switched to this. If we add more detail to `EXPLAIN` we might want to actually return an `explainPlanInfo` struct. Do you care strongly about this now?